### DISCUSSION TITLE
I can't use RPM Measure with mqtt

### DISCUSSION THREAD

**Discussion by jaytnw** [khoih-prog/ESP32_New_TimerInterrupt#3] [General]
I wana publish rpm to broker but it reboot all the time when read HIGH. What should I do? please.

**Comment by khoih-prog** [comment]
There is some issue in your code causing crash. Without [**MRE**](https://stackoverflow.com/help/minimal-reproducible-example), I don't think anybody can have the **good crystal ball** to know where the issue is.

Anyway, the basic rule is not to do complex and time-consuming jobs inside **ISR**. Have a look at the example [**RPM_Measure.ino**#L149-L157](https://github.com/khoih-prog/ESP32_New_TimerInterrupt/blob/main/examples/RPM_Measure/RPM_Measure.ino#L149-L157) and see how to display / publish RPM inside the loop()

Also check [**Important Notes**](https://github.com/khoih-prog/ESP32_New_TimerInterrupt#important-notes)

Please ask for help in Arduino or ESP32 forum because this is **not caused by a bug of this library.**
